source/manage-data.txt

```diff
@@ -0,0 +1,20 @@
+=======================
+Interact with Your Data
```

Comment:
Perhaps not as part of this ticket, but in the future, we should consider:

- Finding a different, consolidated home in the Compass docs for the "Databases" page, the "Collections and Views" pages, and the "Manage Documents" pages (just a thought).

- Promoting all of the pages under the "Query Your Data" and "View Past Queries" expandable subsections to be "top level" pages, meaning that the final version of this ToC section would be "Interact with Your Data" followed by "Set Which Fields Are Returned" down through "Set Validation Rules for Your Schema". Under this organization, there'd be no need for expandable subsections with "Interact with Your Data".

Again, not as part of this ticket, but perhaps something to consider in the future so that we can (1) continue to improve the organization of the docs whenever possible (based on what users are doing with Compass) and (2) highlight a "task-oriented" experience in the docs as much as possible. The Compass docs have so much potential to be _the_ model of what task-oriented doc'n looks like (these docs are doing an amazing job right now) and I'd like to evangelize them as much as possible.